generators/client/templates/src/main/webapp/_index.html

```diff
@@ -76,7 +76,7 @@
         </div>
 
         <div class="footer" ng-cloak>
-            <p data-translate="footer">This is your footer</p>
+            <p class="text-center" data-translate="footer">This is your footer</p>
```

Comment:
I understand that. 
My rationale behind adding it was based on other places in the application jhipster is providing some sort of minimalistic styling hence I thought it would be nice have the footer centered (which is the case in most of the websites)

Nevertheless, I'll remove this.